apps/client/src/app/app.component.ts

```diff
@@ -108,7 +108,9 @@ export class AppComponent implements OnDestroy, OnInit {
     this.materialCssVarsService.setDarkTheme(isDarkTheme);
 
     window.matchMedia('(prefers-color-scheme: dark)').addListener((event) => {
-      this.materialCssVarsService.setDarkTheme(event.matches);
+      if (!this.user?.settings.appearance) {
+        this.materialCssVarsService.setDarkTheme(event.matches);
```

Comment:
I have found this edge case 😄 It should only override the theme if the user has no appearance setting set.

Reply:
Ahh, my bad! Nice catch btw